src/test/java/org/folio/rest/api/TestBase.java

```diff
@@ -65,6 +74,92 @@ public abstract class TestBase {
   static AuthorityReindexFixture authorityReindex;
   static AsyncMigrationFixture asyncMigration;
 
+  @BeforeClass
+  public static void testBaseBeforeClass() {
```

Comment:
Given that you've gone to active effort to rename the before and after methods in many of these. I think it's worth choosing either `all` or `class` as the postfix for once per class methods.

Reply:
Done. I've chosen `all` over `class` to make it more consistent with junit5 (which uses `all` instead of `class`).

Reply:
I very much prefer `all` (and `each`). I only wondered if there was a specific reason this one was different. I know it's hard to track stuff like this when there are so many changes.